include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1572,6 +1572,13 @@ enum {
 };
 
 struct bt_le_per_adv_sync_transfer_param {
+	/**
+	 * @brief Mode
+	 *
+	 * Determines whether or not to sync to the PAST indication, whether or not
+	 * to generate sync reports, and if the reports should be filtered or not.
+	*/
+	uint8_t mode;
```

Comment:
I would suggest to add the mode to the already existing `options` field instead. 

Something along the lines of `BT_LE_PER_ADV_SYNC_TRANSFER_OPT_NO_SYNC` and `BT_LE_PER_ADV_SYNC_TRANSFER_OPT_NO_REPORTS` (and document those options to be mutually exclusive). 

That way it won't be an API change and be fully backwards compatible, as well as following the structure of not only the sync parameters, but the rest of the Bluetooth API.

Reply:
I've added them as flags, with documentation.